addons/test_mail/tests/test_message_track.py

```diff
@@ -22,7 +22,11 @@ def setUpClass(cls):
         ).with_context(cls._test_context).create({
             'name': 'Test',
         })
+        track_record = cls.env['mail.test.track'].with_user(cls.user_employee).with_context(cls._test_context).create({
```

Comment:
As it is used only 'test_message_track_default_message' we can probably create it in that test directly